tests/test_icecream.py

```diff
@@ -183,12 +184,13 @@ def parseOutputIntoPairs(out, err, assertNumLines,
         if len(pairs[0]) == 1 and line.startswith(' '):
             arg, value = linePairs[-1][-1]
             looksLikeAString = value[0] in ["'", '"']
-            prefix = (arg + ': ') + (' ' if looksLikeAString else '')
+            prefix = ((arg + ': ' if arg is not None else '')  # A multiline value
+                      + (' ' if looksLikeAString else ''))
             dedented = line[len(ic.prefix) + len(prefix):]
             linePairs[-1][-1] = (arg, value + '\n' + dedented)
         else:
             items = [
-                (p[0].strip(), None) if len(p) == 1  # A value, like ic(3).
+                (None, p[0].strip()) if len(p) == 1  # A value, like ic(3).
```

Comment:
This caused some change in the tests.
I changed the semantics to be consistent for values and variables so that the pair is always of the format `(arg, value)` (it used to be `(value, arg=None)` before the change). This gives us the multiline handling of values for free since the pairs hold to the semantics.